modules/core/04-channel/keeper/migrations.go

```diff
@@ -16,10 +16,6 @@ func NewMigrator(keeper *Keeper) Migrator {
 	return Migrator{keeper: keeper}
 }
 
-// MigrateParams migrates params to the default channel params.
-func (m Migrator) MigrateParams(ctx sdk.Context) error {
-	params := channeltypes.DefaultParams()
-	m.keeper.SetParams(ctx, params)
-	m.keeper.Logger(ctx).Info("successfully migrated ibc channel params")
-	return nil
+func (m *Migrator) Migrate7To8(ctx sdk.Context) error {
+	return v10.MigrateStore(ctx, m.keeper.storeService, m.keeper.cdc, m.keeper)
```

Comment:
Why are you doing v10 migrate store in this function?

Reply:
This is to have a migrator for channel, but keep the migration logic out of the keeper itself. We have the same process in 02-client:
```
func (m Migrator) Migrate2to3(ctx sdk.Context) error {
	return v7.MigrateStore(ctx, m.keeper.storeService, m.keeper.cdc, m.keeper)
}
```

Reply:
Yes but wouldn't this be called `Migrate8To10`?

Reply:
That number refers to the consensus version of the store. It's a bit of a weird mix. Not sure what the best solution is. But client also mixes it the same way.